.github/CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 <h3>New features since last release</h3>
 
+* A new differentiation method has been added for use with simulators in tape mode. The `"adjoint"`
+  method operates after a forward pass by iteratively applying the inverse gate to scan backwards
+  through the circuit. This method is similar to the reversible method, but has a lower time
+  overhead and a similar memory overhead. It follows the approach provided by
+  [Jones and Gacon](https://arxiv.org/abs/2009.02823). This method is only compatible with certain
+  statevector-based devices such as `default.qubit`.
```

Comment:
needs certain types of methods: `_apply_operation` and `_apply_unitary`, as well as access to `dev._state`.  Now it is written into `QubitDevice` itself, so it at least has to inherit from `QubitDevice` and have those certain functions.

Reply:
Yes, inheriting from `QubitDevice` is not sufficient (and probably never will be, since this could include a hardware device).

But the idea is to make it widely available on most devices.